src/dbally/views/sqlalchemy_base.py

```diff
@@ -63,16 +67,16 @@ async def _build_filter_bool_op(self, bool_op: syntax.BoolOp) -> sqlalchemy.Colu
 
     def execute(self, dry_run: bool = False) -> ExecutionResult:
         """
-        Executes the generated SQL query and returns the results.
-
-        :param dry_run: If True, only generate the query without executing it
+        Executes the generated SQL query and returns the results. Be aware that before running\
```

Comment:
You don't need to. If you run `apply_filters` that the query will execute with those filters applied, if you run it without `apply_filters` that the query will also successfully execute, without applying any filters. Seems to me like common sense.